types/react-navigation/index.d.ts

```diff
@@ -845,7 +846,8 @@ export interface NavigationContainerProps<S = {}, O = {}> {
 
 export interface NavigationContainerComponent
   extends React.Component<
-  NavigationContainerProps & NavigationNavigatorProps<any>
+  NavigationContainerProps & NavigationNavigatorProps<any>,
+  { nav: NavigationState | null }
```

Comment:
Wouldn't it be better to introduce a `NavigationContainerState` interface which looks like this?

```ts
interface NavigationContainerState {
    nav?: NavigationState;
}
```

Reply:
I just aligned on how was declared the state inside NavigationContainer. Do you want me to change only one, both or none?

Reply:
In that case I'll leave that to you, but I would change
`nav: NavigationState | null`
to
`nav?: NavigationState`
for the people who have strictNullCheck enabled